packages/smart-wallet/src/smartWallet.js

```diff
@@ -1019,6 +1058,44 @@ export const prepareSmartWallet = (baggage, shared) => {
           await E(seatRef).tryExit();
         },
       },
+
+      invoke: {
+        /**
+         * @param {InvokeEntryMessage} message
+         */
+        async invokeEntry(message) {
```

Comment:
I kinda wish we had an `invocationId` in the action so that we can publish the outcome of the invocation. Right now it's basically impossible to know when an invocation action has completed without the callee itself publishing something.

Reply:
yes, that's a known limitation. perhaps it's insufficiently documented?

When we use this replace ymax or prune vstorage, vstorage will get updated as a consequence.
I suppose an upgrade doesn't publish. I don't think that's a problem during ymax beta.